.vscode/settings.json

```diff
@@ -1,5 +1,7 @@
 {
-  "editor.rulers": [80],
+  "editor.rulers": [
```

Comment:
This is the most annoying thing.... so when I fixed this, then saved, of course it incorrectly reformatted the fixed file itself!! I keel you vscode

Reply:
`git checkout master .vscode/settings.json`